test/unit/objects/test_model.py

```diff
@@ -656,7 +682,8 @@ def test_bulk_fill_hprobs_with_high_smallness_threshold(self):
     def test_iter_hprobs_by_rectangle(self):
         self.skipTest("TODO should probably warn user?")
 
-
+class LindbladModelTester(GLNDModelBase, StandardMethodBase, BaseCase):
+    pass
```

Comment:
Why is this needed / what is it accomplishing?

Reply:
The testing code (a lot of which is architected like this) is kind of a pain to read. The way to parse this is that a set of generically defined testing methods (generic in the sense of they make sense independent of a model's parameterization) defined in `StandardMethodBase` should be run against a model with parameterization as defined in `GLNDModelBase`, with access to a number of useful testing utilities defined in `BaseCase`. So while it doesn't look like the class definition does anything, it is actually the case that the act of wrapping all of those parent classes as done here (and with particular keywords in the class name) results in a test suite which pytest knows how to parse and run (I think it looks for "Tester" in the class name, among other possible keywords).

Reply:
Gotcha. Marking as resolved!